ratethisapp/src/main/java/com/kobakei/ratethisapp/RateThisApp.java

```diff
@@ -178,7 +164,8 @@ public static boolean shouldShowRateDialog() {
      * @param context
      */
     public static void showRateDialog(final Context context) {
-        AlertDialog.Builder builder = new AlertDialog.Builder(context);
```

Comment:
How about `builder` and `dialog` are not null? (= dialog is shown)
It may be better to do nothing on the situation.